From: Simon McVittie <smcv@debian.org>
Date: Thu, 15 Apr 2021 13:41:44 +0100
Subject: Disable web fonts for now

The way this works is still changing upstream, and getting the
correct licensing and source code for the Red Hat font (ITP: #986999)
is not really in the scope of packaging gi-docgen-dependent projects.

Forwarded: not-needed, Debian-specific
---
 .../gi-docgen/gidocgen/templates/basic/fonts.css   | 176 ---------------------
 .../gi-docgen/gidocgen/templates/basic/style.css   |   2 -
 2 files changed, 178 deletions(-)

diff --git a/subprojects/gi-docgen/gidocgen/templates/basic/fonts.css b/subprojects/gi-docgen/gidocgen/templates/basic/fonts.css
index ce790e7..0a01b93 100644
--- a/subprojects/gi-docgen/gidocgen/templates/basic/fonts.css
+++ b/subprojects/gi-docgen/gidocgen/templates/basic/fonts.css
@@ -3,179 +3,3 @@
  *
  * SPDX-License-Identifier: Apache-2.0 OR GPL-3.0-or-later
  */
-
-/**
- * RedHat Fonts taken from https://github.com/RedHatOfficial/RedHatFont
- * License: SIL Open Font License 1.1 http://scripts.sil.org/OFL
- *
- * SourceCodePro Fonts
- * License: SIL Open Font License 1.1 https://scripts.sil.org/OFL
- */
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Regular.woff2") format("woff2"),
-       url("RedHatDisplay-Regular.woff") format("woff");
-  font-style: normal;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-RegularItalic.woff2") format("woff2"),
-       url("RedHatDisplay-RegularItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Medium.woff2") format("woff2"),
-       url("RedHatDisplay-Medium.woff") format("woff");
-  font-style: normal;
-  font-weight: 500;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-MediumItalic.woff2") format("woff2"),
-       url("RedHatDisplay-MediumItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 500;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Bold.woff2") format("woff2"),
-       url("RedHatDisplay-Bold.woff") format("woff");
-  font-style: normal;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-BoldItalic.woff2") format("woff2"),
-       url("RedHatDisplay-BoldItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Black.woff2") format("woff2"),
-       url("RedHatDisplay-Black.woff") format("woff");
-  font-style: normal;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-BlackItalic.woff2") format("woff2"),
-       url("RedHatDisplay-BlackItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Regular.woff2") format("woff2"),
-       url("RedHatText-Regular.woff") format("woff");
-  font-style: normal;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Italic.woff2") format("woff2"),
-       url("RedHatText-Italic.woff") format("woff");
-  font-style: italic;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Medium.woff2") format("woff2"),
-       url("RedHatText-Medium.woff") format("woff");
-  font-style: normal;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-MediumItalic.woff2") format("woff2"),
-       url("RedHatText-MediumItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Bold.woff2") format("woff2"),
-       url("RedHatText-Bold.woff") format("woff");
-  font-style: normal;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-BoldItalic.woff2") format("woff2"),
-       url("RedHatText-BoldItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-/* Do not fall back to locally installed SourceCodePro, as there are
- * known bad copies being distributed to this day
- */
-@font-face {
-  font-family: 'Source Code Pro';
-  font-style: normal;
-  font-weight: 400;
-  src: url("SourceCodePro-Regular.ttf.woff") format('woff');
-  font-display: swap;
-}
-
-@font-face {
-  font-family: 'Source Code Pro';
-  font-style: italic;
-  font-weight: 400;
-  src: url("SourceCodePro-It.ttf.woff") format('woff');
-  font-display: swap;
-}
-
-@font-face {
-  font-family: 'Source Code Pro';
-  font-style: normal;
-  font-weight: 600;
-  src: url("SourceCodePro-Semibold.ttf.woff") format('woff');
-  font-display: swap;
-}
diff --git a/subprojects/gi-docgen/gidocgen/templates/basic/style.css b/subprojects/gi-docgen/gidocgen/templates/basic/style.css
index 188c9a7..954b5d8 100644
--- a/subprojects/gi-docgen/gidocgen/templates/basic/style.css
+++ b/subprojects/gi-docgen/gidocgen/templates/basic/style.css
@@ -7,8 +7,6 @@
 @import url("solarized-light.css") (prefers-color-scheme: light);
 @import url("solarized-dark.css") (prefers-color-scheme: dark);
 
-@import url("fonts.css");
-
 /*********************************
  * LIGHT THEME
  *********************************/
